fahadmak/Nai-Java-Devs
# #155820624 Launch DetailActivity from MainActivity

[Delivers #155820624]
#### What does this PR do?
- [x] This PR enables Launching the DetailActivity from the MainActivity

#### Description of Task to be completed?
- [x] Create the UsersAdapter.java file
- [x] Create an instance ListView in oncreate method
- [x] Create an instance UserAdapter in oncreate method and pass the UserAdapter instance  as an argument in the setAdapter method of Listview instance.
- [x] Add a setOnItemClickListener in onCreate method
- [x] Build an Intent in setOnItemClickListener method
- [x] Pass the intent in startActivity() method
- [x] Get the intent in the onCreate method and extra data passed in the intent
- [x] Capture the activity_detail.xml layout's TextView and ImageView and set the string in the intent's data and image in the intent's data respectively.
#### How should this be manually tested?
- Clone the repo and cd into it
- Checkout to origin/ft-launch-DetailActivity-155820624
- Open the project in Android Studio
- Run the android emulator and click a list item in the MainActivity shown in the emulator
- Verify that DetailActivity is started that shows a single user profile

#### What are the relevant pivotal tracker stories?
[#155820624](https://www.pivotaltracker.com/story/show/155820624)

#### Screenshots (if appropriate)
##### MainActivity
<img width="200" alt="Screen Shot 2019-03-25 at 6 28 43 PM" src="https://user-images.githubusercontent.com/13882936/54932463-ed9a2100-4f2b-11e9-9196-93fb93986b80.png">

##### DetailActivity
<img width="200" alt="Screen Shot 2019-03-25 at 6 28 32 PM" src="https://user-images.githubusercontent.com/13882936/54932552-0c001c80-4f2c-11e9-8975-dd9c76d2e279.png">
